docs/sphinx/source/pvsystem.rst

```diff
@@ -0,0 +1,165 @@
+.. _pvsystemdoc:
+
+PVSystem
+========
+
+.. ipython:: python
+    :suppress:
+
+    import pandas as pd
+    from pvlib import pvsystem
+
+The :py:class:`~pvlib.pvsystem.PVSystem` class wraps many of the
+functions in the :py:mod:`~pvlib.pvsystem` module. This simplifies the
+API by eliminating the need for a user to specify arguments such as
+module and inverter properties when calling PVSystem methods.
+:py:class:`~pvlib.pvsystem.PVSystem` is not better or worse than the
+functions it wraps -- it is simply an alternative way of organizing
+your data.
+
+This guide aims to build understanding of the PVSystem class. It assumes
+basic familiarity with object-oriented code in Python, but most
+information should be understandable without a solid understanding of
+classes. Keep in mind that `functions` are independent of objects,
+while `methods` are attached to objects.
+
+See :py:class:`~pvlib.modelchain.ModelChain` for an application of
+PVSystem to time series modeling.
+
+
+.. _designphilosophy:
+
+Design philosophy
+-----------------
+
+The PVSystem class is designed to separate the data that represents a PV
+system (e.g. tilt angle or module parameters) from the data that
+influences the PV system (e.g. the weather).
+
+The data that represents the PV system is *intrinsic*. The
+data that influences the PV system is *extrinsic*.
+
+Intrinsic data is stored in object attributes. For example, the data
+that describes a PV system's module parameters is stored in
+`PVSystem.module_parameters`.
+
+.. ipython:: python
+
+    module_parameters = {'pdc0': 10, 'gamma_pdc': -0.004}
+    system = pvsystem.PVSystem(module_parameters=module_parameters)
+    print(system.module_parameters)
+
+Extrinsic data is passed to a PVSystem as method arguments. For example,
+the :py:meth:`~pvlib.pvsystem.PVSystem.pvwatts_dc` method accepts extrinsic
+data irradiance and temperature.
+
+.. ipython:: python
+
+    pdc = system.pvwatts_dc(1000, 30)
+    print(pdc)
+
+Compare the :py:meth:`~pvlib.pvsystem.PVSystem.pvwatts_dc` method signature
```

Comment:
Add text at the beginning:   `Methods attached to a PVSystem object wrap corresponding functions in pvsystem.py.  The methods simplify the argument list by using data stored in the PVSystem attributes.`